### DISCUSSION TITLE
Glossary

### DISCUSSION THREAD

**Discussion by cdes** [tolgee/tolgee-platform#1417] [Ideas]
I believe it's an essential feature for localization platforms. For example, a word may have several meanings or translations, meaning that a translator will have to consider those options each time they translate that word. A glossary provides an already-approved term, a translation, and an explanation to ensure correct usage.

There are common types of words or phrases added to a glossary:
- Brand or company-specific phrases
- Industry-specific terms or phrases
- Commonly used terms
- Forbidden terms
- Non-translatables
- Abbreviations.

Examples:
https://docs.lokalise.com/en/articles/1400629-glossary
https://help.transifex.com/en/articles/6240837-using-the-glossary
https://localazy.com/features/glossary
https://help.smartling.com/hc/en-us/articles/115004844973--Introduction-to-the-Glossary

**Comment by JanCizmar** [comment]
Hi there! Thanks for letting us know. The Glossary feature is on our [roadmap](https://tolgee.io/roadmap). And we plan to implement it next year. I've also created an [issue in tolgee-platform repo](https://github.com/tolgee/tolgee-platform/issues/1418).
